scripts/install_narrative_docker.sh

```diff
@@ -82,6 +100,13 @@ jupyter nbextension enable widgetCell/main --sys-prefix
 jupyter nbextension install $(pwd)/dataCell --symlink --sys-prefix
 jupyter nbextension enable dataCell/main --sys-prefix
 
+jupyter nbextension install $(pwd)/editorCell --symlink --sys-prefix
```

Comment:
These extra `jupyter nbextension install` blocks reference cell types that don't exist in staging - editorCell and appCell2 should only be in develop right now.